plane/parse_xml.py

```diff
@@ -3,15 +3,16 @@
 from scipy.linalg import polar
 
 class ParseXML:
-    def __init__(self, source_points=None, dest_points=None, M=None, z=0):
+    def __init__(self, source_points=None, dest_points=None, M=None, z=0, template_id = None):
```

Comment:
Please provide a more descriptive name to the class ParseXML is a bit generic.

Reply:
I was going to rename it to FitPlane, but as per one of your other comments if we eventually rename FitMultiPlane to FitPlane to match coregister-xz, this class needs a different name.

Reply:
Maybe FitTemplate?